app/javascript/src/legacy/form_builder_survey.js

```diff
@@ -25,9 +25,10 @@
       // that.copy_in_follow_up_dependency(this);
     });
 
-    $(document).on('nested:fieldRemoved', function(event) {
-      confirm('Please submit the form to confirm your changes.');
-    });
+    // $(document).on('nested:fieldRemoved', function(event) {
+    //   console.log("I AM IN THIS JS FUNCTION.")
+    //   confirm('Please submit the form to confirm your changes.');
+    // });
```

Comment:
might still need this? reminds the user to save. if not needed, remove the commented function